test/factories/users.rb

```diff
@@ -16,6 +16,10 @@
       Rack::Test::UploadedFile.new(tempfile.path, 'image/png')
     end
 
+    after(:build) do |user|
+      user.data.email_status = :verified
+    end
```

Comment:
Setting `email_status { :verified }` in the `User::Data` factory didn't work, so 🤷 
I had a nicer solution, but that broke some other tests.